docs/blockbuilder.md

```diff
@@ -17,6 +17,8 @@
       - [Custom blockquote](#custom-blockquote)
   - [Extending the package WPImportStreamBlocks](#extending-the-package-wpimportstreamblocks)
 
+`All code examples are for a site using Wagtail v3.0+` See [Wagtail release notes](https://docs.wagtail.org/en/stable/releases/3.0.html) for compatibility for Wagtail versions <3.0
```

Comment:
**Suggestion**: Don't use backticks for prose, just code.

Reply:
Thanks Nick: Fixed